mlir-tensorrt/executor/lib/Runtime/Backend/Lua/Modules/TensorRT/TensorRTModule.cpp

```diff
@@ -37,6 +37,14 @@
 #pragma GCC diagnostic pop
 #endif
 
+/// Evaluates to true if the version of TensorRT that we are compiling against
+/// is greater than or equal to the given version number.
+#define TENSORRT_MODULE_VERSION_GTE(major, minor, patch)                       \
```

Comment:
Can we use `MLIR_TRT_COMPILE_TIME_TENSORRT_VERSION_GTE` that is already available?

Reply:
I was running into some compilation issues while including `mlir-tensorrt/tensorrt/include/mlir-tensorrt-dialect/Utils/TensorRTVersion.h` here.
I did not investigate the compilation issue further, but does that dependency make sense in the `mlir-tensorrt/executor`? If yes, I can update the Makefile to add the dependency.

Reply:
This should depend only on `MLIRTRTTensorRTDynamicLoader`, I think. Adding dependency is okay.

Reply:
I think I needed to explicitly include a header directory. Link time dependency is not required.